### PR TITLE
Add GIF for Mac latest

### PR DESCRIPTION
**Pull request by remko**

### CODE REVIEW

File: Casks/gif-for-mac.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'gif-for-mac' do
+  version :latest
+  sha256 :no_check
+
+  url "https://d309cd6updicdi.cloudfront.net/mac/bin/GIFforMac.dmg"
```
Comment (vitorgalvao) [inline Casks/gif-for-mac.rb]: Double quotes are reserved for cases where string interpolation is needed. Please change these to single quotes.

Comment (vitorgalvao) [inline Casks/gif-for-mac.rb]: Needs a `# cloudfront.net is the official download host per the vendor homepage` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/cb487ad405f74cb1f26b93e3f3e8412c82a22e34/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/7c010b42bd7ac679c1f002a0368b36bb3f495612/doc/CASK_LANGUAGE_REFERENCE.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

File: Casks/gif-for-mac.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  name 'GIF for Mac'
+  homepage 'https://www.riffsy.com/Mac'
+  license :unknown
```
Comment (vitorgalvao) [inline Casks/gif-for-mac.rb]: `:gratis`.

**Comment by victorpopkov** [comment]
Thank you for the contribution. It was merged directly as cb7e97f5ecef6ec4828e91640556966705662224 to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**